app/models/katello/glue/pulp/node.rb

```diff
@@ -0,0 +1,23 @@
+module Katello
+  module Glue::Pulp::Node
```

Comment:
I discussed and checked with @jlsherrill and we should be good using OAuth credentials. At least for now. There's rework to use certificates coming, but it's independent of this PR.